ui/app/common/displaycontrols/admissiondetails/directives/admissionDetails.js

```diff
@@ -37,15 +37,17 @@ angular.module('bahmni.common.displaycontrol.admissiondetails')
             };
         };
         var calculateDaysAdmitted = function ($scope) {
-            if ($scope.visitSummary.admissionDetails && $scope.visitSummary.dischargeDetails) {
-                var admissionDate = new Date($scope.visitSummary.admissionDetails.date);
-                var dischargeDate = new Date($scope.visitSummary.dischargeDetails.date);
-                var timeDifference = dischargeDate - admissionDate;
-                var daysAdmitted = Math.ceil(timeDifference / (1000 * 60 * 60 * 24));
-                $scope.visitSummary.daysAdmitted = daysAdmitted;
-                $scope.visitSummary.showDaysAdmitted = true;
-            } else {
-                $scope.visitSummary.showDaysAdmitted = false;
+            if ($scope.visitSummary) {
+                if ($scope.visitSummary.admissionDetails && $scope.visitSummary.dischargeDetails) {
```

Comment:
You can add in single line --> $scope.visitSummary  && ...

Reply:
Tried that, but the visitSummary itself is undefined at this time. Due to which the else will throw an error (showDaysAdmitted scope variable)

Reply:
Okay 👍